src/spikeinterface/sorters/external/mountainsort5.py

```diff
@@ -68,8 +67,8 @@ class Mountainsort5Sorter(BaseSorter):
         "freq_max": "Low-pass filter cutoff frequency",
         "filter": "Enable or disable filter",
         "whiten": "Enable or disable whitening",
-        "temporary_base_dir": "Temporary directory base directory for storing cached recording",
         "n_jobs_for_preprocessing": "Number of parallel jobs for creating the cached recording",
+        "delete_temporary_recording": "If True, the temporary recording file is deleted after sorting",
```

Comment:
```suggestion
        "delete_temporary_recording": "If True, attempt to delete the temporary recording file after sorting (this may fail on Windows requiring the end-user to delete the file themselves later",
```